db/migrate/20170713021445_create_invites.rb

```diff
@@ -0,0 +1,14 @@
+class CreateInvites < ActiveRecord::Migration[5.1]
+  def change
+    enable_extension 'pgcrypto'
```

Comment:
Now that I think about, we should probably put this in a different migration so it's obvious which migration added the extension.